nimble/transport/apollo3/syscfg.yml

```diff
@@ -0,0 +1,37 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+syscfg.defs:
+    BLE_TRANSPORT_APOLLO3_HCI_DRV_MAX_TX_PACKET:
+        description: >
+            Maximum TX packet size to controller in bytes.
+        value: 256
+
+    BLE_TRANSPORT_APOLLO3_HCI_DRV_MAX_RX_PACKET:
+        description: >
+            Maximum RX packet size from controller in bytes.
+        value: 256
+
+    BLE_TRANSPORT_APOLLO3_HCI_DRV_MAX_XTAL_RETRIES:
```

Comment:
oh, I should be more specific - I thought actually about calling this simply `BLE_TRANSPORT_APOLLO3_MAX_XTAL_RETRIES` (and similar for other settings), but it can be left as it is now - up to you